addons/mail/models/discuss/discuss_channel.py

```diff
@@ -784,7 +786,8 @@ def _notify_thread(self, message, msg_vals=False, **kwargs):
             payload["temporary_id"] = temporary_id
         if kwargs.get("silent"):
             payload["silent"] = True
-        Store(self, {"is_pinned": True}, bus_channel=self, bus_subchannel="members").bus_send()
+        for member in self.channel_member_ids:
+            Store(member, "is_pinned", bus_channel=member._bus_channel()).bus_send()
```

Comment:
Now that I think about it, this line makes little sense here.
It should be moved to where we set the `last_interest_dt` value instead.

And also, now that I think about it too, we should not be sending `is_pinned` at all to the client, this would resolve most if not all issues in regards to number of bus notif.

We should just be sending `last_interest_dt` (channel) when it changes, which can still be done on the channel bus. (It's already done in `_sync_field_names` actually).

And `is_pinned` should be computed in JS similarly to how it is computed in python. Making sure we do send properly all the necessary fields, both `last_interest_dt` of channel and member (this should already be the case I think) and `unpin_dt`.